jwt.go

```diff
@@ -384,6 +395,44 @@ func (jwtPlugin *JwtPlugin) ExtractToken(request *http.Request) (*JWT, error) {
 	return &jwtToken, nil
 }
 
+func (jwtPlugin *JwtPlugin) remoteAddr(req *http.Request) Network {
+	// This will only be defined when site is accessed via non-anonymous proxy
+	// and takes precedence over RemoteAddr
+	// Header.Get is case-insensitive
+	ipHeader := req.Header.Get("X-Forwarded-For")
```

Comment:
Does X-Forwarded-For actually contain the port as well? I guess it doesn't matter as the port is not the most relevant thing.